Frontend/Controllers/Projects/FeaturesController.cs

```diff
@@ -162,57 +165,45 @@ public async Task<IActionResult> Type(string urn, bool returnToPreview = false)
                 return View("ErrorPage", project.Error.ErrorMessage);
             }
 
-            var model = new FeaturesViewModel
+            var vm = new FeaturesTypeViewModel
             {
-                Project = project.Result,
+                Urn = project.Result.Urn,
+                OutgoingAcademyName = project.Result.OutgoingAcademyName,
                 ReturnToPreview = returnToPreview
             };
-            return View(model);
+            return View(vm);
```

Comment:
If you go to the type of transfer page of an existing project that has the Type of transfer value set, it doesn't set the radio button, this may be to do with this - e.g. not sure if you need to also populate the type of transfer in the vm, but give me a shout if you want me to show you what I mean